tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -207,6 +207,10 @@ def check_roslaunch(f):
         file_deps = {}
     for pkg, miss in missing.items():
         if miss:
+            print("Missing package dependencies: %s/package.xml: %s"%(pkg, ', '.join(miss)))
```

Comment:
Why should this message be still printed? I thought the point was that this is not actually a problem?